### PR TITLE
Fix: Fixed issue where the prompt to review would crash the app

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Fixed issue where the prompt to review would crash the app [636092730u](https://appcenter.ms/orgs/FilesApp/apps/Files/crashes/errors/636092730u/overview)

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: src/Files.App/Views/MainPage.xaml.cs [hunk 1]
```diff
@@ -328,6 +324,17 @@ private void Page_Loaded(object sender, RoutedEventArgs e)
 			FindName(nameof(InnerNavigationToolbar));
 			FindName(nameof(horizontalMultitaskingControl));
 			FindName(nameof(NavToolbar));
+
+			// Prompt user to review app in the Store
+			if
+			(
+				SystemInformation.Instance.TotalLaunchCount >= 15 &
+				Package.Current.Id.Name == "49306atecsolution.FilesUWP" &&
```
Comment (QuaintMako) [inline src/Files.App/Views/MainPage.xaml.cs:332]: Extracting the string value inside a constant?

Comment (yaira2) [reply src/Files.App/Views/MainPage.xaml.cs:332]: Is there a constant?

Comment (flaviabanchieri) [reply src/Files.App/Views/MainPage.xaml.cs:332]: > Extracting the string value inside a constant?

I think there's no constant, or, case there's one, where is the constant?

Comment (yaira2) [reply src/Files.App/Views/MainPage.xaml.cs:332]: Looks like this is the only place where we use the package name so we can keep as is.

**Review by QuaintMako — APPROVED** [review]
LGTM.

**Pull request merged**